src/Settings.php

```diff
@@ -380,7 +380,7 @@ public function hooks() {
 		add_filter( 'cmb2_render_pagetype_select_taxonomy', [ $this, 'p4_render_pagetype_dropdown' ], 10, 2 );
 
 		// Make settings multilingual if wpml plugin is installed and activated.
-		if ( function_exists( 'icl_object_id' ) ) {
+		if ( is_plugin_active( 'sitepress-multilingual-cms/sitepress.php' ) ) {
```

Comment:
Tests are failing because the file `wp-admin/includes/plugin.php` containing the function `is_plugin_active` is not included during tests. We're already including it in [gutenberg plugin](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/18c2a81ec9f4e0a0057d817c9a40607447717d97/planet4-gutenberg-blocks.php#L101), maybe we should do the same for theme ? I don't know if the same problem could happen on the frontend or only on tests.

Reply:
The easiest workaround I pick is check `function_exists()` 😉